modified footer enhancement

Soumyosish opened the pull request:
I have implemented the footer enhancement again. Please @Amarjha01 check and merge the PR. 
<img width="1909" height="669" alt="image" src="https://github.com/user-attachments/assets/8912bcbc-2cff-4463-84f1-dd24c7612673" />
Fixes #29

Soumyosish committed:
modified footer enhancement

Soumyosish committed:
Merge branch 'master' of https://github.com/Soumyosish/InfantCareCompass into modified-footer-enhancement

Soumyosish committed:
commit

CHHemant reviewed (changes requested):
@Amarjha01 @Soumyosish It's good, but somewhere it's not matching the color of the page, so please check that and make the changes. It should be some purple and blue and match the page. so it can look better. I appreciate your work, but there should be some changes required.

Soumyosish committed:
modified enhanced footer changed

Soumyosish commented:
<img width="1885" height="603" alt="image" src="https://github.com/user-attachments/assets/f4a51e9e-a48f-410c-a68e-a6b6cf0846e0" />
I have changed the heading of the footer to blue style along with purple effect as every button is having that effect in the website along with that also changed the back to top icon align that color. Please check @Amarjha01 @CHHemant .

Amarjha01 commented:
Thanks @Soumyosish  for contributing to InfantCareCompass! 👶🩺
Your efforts are helping us build a smarter and more supportive platform for parents and caregivers.
We truly appreciate your time and contribution — keep up the great work! 🚀

I'm currently a bit tied up, but I’ll review the code and merge it as soon as I get a chance. 😊

Amarjha01 commented:
Thanks  @Soumyosish  for contributing to InfantCareCompass! 👶🩺
Your work helps us build a smarter, more helpful platform for parents and caregivers.
We appreciate your time and effort — keep making an impact! 🚀
